sor/src/main/java/com/bazaarvoice/emodb/sor/db/astyanax/AstyanaxDataReaderDAO.java

```diff
@@ -292,14 +291,14 @@ public Record read(Key key, ReadConsistency consistency) {
         ByteBuffer rowKey = storage.getRowKey(key.getKey());
 
         // Read Delta and Compaction objects
-        Iterator<Change> deltas = Iterators.emptyIterator();
+        Iterator<Change> deltas = Collections.emptyIterator();
         if (includeContentData) {
             ColumnFamily<ByteBuffer, UUID> cf = placement.getDeltaColumnFamily();
             deltas = decodeColumns(columnScan(rowKey, placement, cf, start, end, reversed, limit, 0, consistency));
         }
 
         // Read History objects
-        Iterator<Change> deltaHistory = Iterators.emptyIterator();
+        Iterator<Change> deltaHistory = Collections.emptyIterator();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "deltaHistory". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS1854)